ytfzf

```diff
@@ -571,7 +571,31 @@ set_vars () {
     #this comes from invidious' api
     thumbnail_quality="high"
     sub_link_count="2"
-    invidious_instance="https://y.com.sb" yt_video_link_domain="https://youtube.com"
+
+    # URL for instance list sorted by type and health
+    instances_url="https://api.invidious.io/instances.json?sort_by=type,health" 
+
+    # Cache location.
+    # TODO: decide on cache interval to avoid instances.json becoming stale
+    instances_file="$cache_dir/instances.json"
```

Comment:
I also tried my hand at implementing your idea, and I avoided having an interval because it sounded messy, what I did was create a `--refresh-inv-instances` option that just refreshes the cache.

However if you think that it can be done clearly, I'm open ears.

Reply:
Your idea sounds much more sensible. It allows us to entirely sidestep having to arrive at a properly motivated caching interval, which is much cleaner.